pyfr/solvers/euler/kernels/bcs/char-riem-inv.mako

```diff
@@ -2,11 +2,14 @@
 <%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
 
 <% gmo = c['gamma'] - 1.0 %>
-<% gamma, v = c['gamma'], c['v'] %>
-<% cs, s = (c['gamma']*c['p']/c['rho'])**0.5, c['p']/(c['rho']**c['gamma']) %>
-<% ratio = 2.0*(c['gamma']*c['p']/c['rho'])**0.5/(c['gamma'] - 1.0) %>
+<% gamma = c['gamma'] %>
 
 <%pyfr:macro name='bc_rsolve_state' params='ul, nl, ur, ploc, t'>
+    <% v = [c[i] for i in 'uvw'[:ndims]] %>
+    fpdtype_t cs = sqrt(${gamma}*${c['p']}/${c['rho']});
+    fpdtype_t s = ${c['p']}/pow(${c['rho']}, ${gamma});
```

Comment:
Missing parenthesis.  I am also not sure why we're dividing instead of multiplying by rho^-gamma.